### ISSUE TITLE
CartoCss Text Height

### ISSUE THREAD

**Issue by lasyakoechlin** [CartoDB/mobile-sdk#371]
Hi,

I am using 4.3.1 on Android.

I am using the GeoJsonDataSource and CartoCss styling. I am using the `building-height` to add 3D, which is working very nicely (see below).

However, I'm having a bit of an issue with the text. Specifically, the text is being drawn below the 3D polygons, so it's giving a different colour depending on if it sticks out or not and that is much less predictable.

I've experimented with setting the `Z` value for the `PointGeometry` and I've experimented with specifying the `building-height` for the `text` in CartoCss (to match the `height` of the polygon that it is associated with). And, neither of those seemed to work.

Basically, I want the text to sit on top of the 3D polygon rather than underneath it... I imagine that will give a much better UI/UX, since it will give consistent text colour.

Do you have any recommendations or suggestions? Thanks!

![image](https://user-images.githubusercontent.com/60749601/85065922-fbb10e80-b17b-11ea-8472-87aeb84d541f.png)

**Comment by mtehver** [comment]
The rendering order of buildings and texts in vector tiles is somewhat tricky and not really based on CartoCSS order. I suggest trying the following snippet:

```
vectorTileLayer.setLabelRenderOrder(VECTOR_TILE_RENDER_ORDER_LAST);
vectorTileLayer.setBuildingRenderOrder(VECTOR_TILE_RENDER_ORDER_LAYER);
```

By default the ordering is 'flipped', that is buildings are rendered on top of everything else.

**Comment by lasyakoechlin** [comment]
That worked, thanks!
